src/Backend/Core/Layout/Templates/footer.html.twig

```diff
@@ -2,7 +2,7 @@
   <div class="container-fluid">
     <div class="row">
       <div class="col-xs-4 col-xs-offset-4 col-sm-4 col-sm-offset-4">
-        <div class="footer-inner">
+        <div class="footer-inner" lang="en">
           <p class="text-center text-muted">
             <small><i>Powered by</i></small>
```

Comment:
Am I correct if there is an image at the end? If so: add "Fork CMS" for screen-readers, so it reads "Powerd by Fork CMS"

Reply:
I thinks it will read Fork CMS, because that is the alt-tag for the image

Reply:
nm, I thought the image was added thru css